modules/gitops-token-storage-and-access.adoc

```diff
@@ -0,0 +1,52 @@
+// Module is included in the following assemblies:
+//
+// * accesscontrol_usermanagement/managing-local-users-in-argo-cd.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-token-storage-and-access_{context}"]
+= Token storage and access
+
+For each local user, the Argo CD Operator creates a Kubernetes secret in the same namespace as the Argo CD instance. The secret name follows the format `{username}-local-user`.
+
+.Example user secret
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: api-user-local-user
+  namespace: argocd
+  labels:
+    app.kubernetes.io/component: local-users
+    app.kubernetes.io/managed-by: argocd-operator
+type: Opaque
+data:
+  apiToken: <base64-encoded-jwt-token>
+  user: <base64-encoded-username>
+  expAt: <base64-encoded-expiration-timestamp>
+  tokenLifetime: <base64-encoded-lifetime-setting>
+  autoRenew: <base64-encoded-auto-renew-setting>
+----
+
+[NOTE]
+====
+The values in the secret are base64-encoded. You must decode them before using.
+====
+
+== Retrieving tokens
+
+To retrieve a user's API token from the Kubernetes secret, run the following command:
+
+[source,terminal]
+----
+kubectl get secret api-user-local-user -n argocd -o jsonpath='{.data.apiToken}' | base64 -d
+----
+
+== Using tokens
+
+To use the retrieved token with the Argo CD CLI, run the following command:  
```

Comment:
Should make it clear this is an example, there are other commands that can be run aside from `app list`

Reply:
We can leave this as is because we want users to stick to an approach. Giving them too many options may get them distracted @jopit.